test/babelish/test_csv2android.rb

```diff
@@ -3,7 +3,8 @@ class TestCSV2Android < Test::Unit::TestCase
 
   def test_converting_csv_to_xml
     csv_file = "test/data/test_data.csv"
-    converter = Babelish::CSV2Android.new(csv_file, 'English' => "en")
+    converter = Babelish::CSV2Android.new(csv_file,
+										  "English" => "en")
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Tab detected.